### ISSUE TITLE
download to local server ? (server to server) 

### ISSUE THREAD

**Issue by yoyoy** [MonstaApps/Monsta-FTP#81]
hello
is it possible with monsta to download files direcly on the server where I install monsta ? 
I would like to copy files from a server to another

thanks ;)

**Comment by MonstaApps** [comment]
Yes, you can download from http(s) URLs using the file fetch feature. It's not possible to download from another FTP server.
